service_test.go

```diff
@@ -304,6 +324,31 @@ func TestUpdateServiceWithSystem(t *testing.T) {
 	autopilot.Equals(t, "Foo", result.Name)
 }
 
+func TestNewUpdateServiceWithFields(t *testing.T) {
+	// Arrange
+	testRequest := autopilot.NewTestRequest(
+		`mutation ServiceUpdate($input:ServiceUpdateInputV2!){serviceUpdate(input: $input){service{apiDocumentPath,description,framework,htmlUrl,id,aliases,language,lifecycle{alias,description,id,index,name},managedAliases,name,owner{alias,id},parent{id,aliases},preferredApiDocument{id,htmlUrl,source{... on ApiDocIntegration{id,name,type},... on ServiceRepository{baseDirectory,displayName,id,repository{id,defaultAlias},service{id,aliases}}},timestamps{createdAt,updatedAt}},preferredApiDocumentSource,product,repos{edges{node{id,defaultAlias},serviceRepositories{baseDirectory,displayName,id,repository{id,defaultAlias},service{id,aliases}}},{{ template "pagination_request" }},totalCount},tags{nodes{id,key,value},{{ template "pagination_request" }},totalCount},tier{alias,description,id,index,name},timestamps{createdAt,updatedAt},tools{nodes{category,categoryAlias,displayName,environment,id,url,service{id,aliases}},{{ template "pagination_request" }},totalCount}},errors{message,path}}}`,
```

Comment:
No `ServiceUpdateInputV2` will also not work because thats not a type in our graphql API.  There is a way to control what is generated here i just don't recall of the top of my head its over in the framework docs.  We can ONLY use `ServiceUpdateInput` here.  But like i said there is a way to override this generation for the go type named `ServiceUpdateInputV2`